ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/SnapProtocol.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.eth;
```

Comment:
If SnapSync is a subprotocol under devp2p, maybe this can go in the p2p package?

Reply:
It's more to do like the EthProtocol class https://github.com/hyperledger/besu/blob/2abc7074bb54ed0c92ad143dfc96166c1f6ac5a4/ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/EthProtocol.java#L33